masonry/src/widget/widget_state.rs

```diff
@@ -90,6 +90,7 @@ pub struct WidgetState {
     pub(crate) needs_accessibility_update: bool,
 
     /// Any descendant has requested an animation frame.
+    pub(crate) needs_anim: bool,
     pub(crate) request_anim: bool,
```

Comment:
I think some differentiating comments for these two fields would be helpful. Looking at the code, it looks like needs_anim allows it to start the update anim pass, and request_anim instructs it to send the lifecycle event during the update anim pass. If so, I think clearer variable names could be helpful.

Reply:
It's not clearly documented yet, but a naming scheme I'm starting to adopt for these flags is that "request_x" means "this widget requested x" and "needs_x" means "this widget or a descendant requested x".

Reply:
I've added some comments in #522. Will add more here later.

Reply:
Done.